addons/mail/static/src/core/common/thread.js

```diff
@@ -153,6 +156,18 @@ export class Thread extends Component {
         onMounted(async () => {
             this.lastJumpPresent = this.props.jumpPresent;
             await this.threadService.fetchNewMessages(this.props.thread);
+            if (this.props.thread.id === "inbox") {
```

Comment:
```js
this.props.thread.eq(this.store.discuss.inbox)
```
Is more robust to check inbox than relying on `id` value. Most threads are managed by discuss team, but if other devs want to make their own threads with id `inbox` then it might clash here